validator/node/node.go

```diff
@@ -264,6 +264,7 @@ func (c *ValidatorClient) initializeFromCLI(cliCtx *cli.Context, router *mux.Rou
 		if isWeb3SignerURLFlagSet {
 			c.wallet = wallet.NewWalletForWeb3Signer()
 		} else {
+			fmt.Println("initializeFromCLI asking for wallet")
```

Comment:
Why a `fmt.Println`?

Reply:
that's a mistake from debugging , should remove